pkgs/applications/editors/neovim/default.nix

```diff
@@ -97,12 +93,12 @@ in
     disallowedReferences = [ stdenv.cc ];
 
     cmakeFlags = [
-      "-DGPERF_PRG=${gperf}/bin/gperf"
-      "-DLUA_PRG=${neovimLuaEnv.interpreter}"
-      "-DLIBLUV_LIBRARY=${luvpath}"
+      # Don't use downloaded dependencies. At the end of the configurePhase one
+      # can spot that cmake says this option was "not used by the project".
+      # That's because all dependencies were found and
+      # third-party/CMakeLists.txt is not read at all.
       "-DUSE_BUNDLED=OFF"
     ]
-    ++ optional doCheck "-DBUSTED_PRG=${neovimLuaEnv}/bin/busted"
```

Comment:
have you checked `make functionaltest` work without this ?

Reply:
Apparently, the `checkPhase` fails with and without this line:

```
neovim-unwrapped> [ RUN      ] user config init loads init.lua from XDG config home by default: -- Tests exited non-zero: Process terminated due to timeout
neovim-unwrapped> -- Output to stderr:
neovim-unwrapped> sh: invalid number '--'
neovim-unwrapped> sh: invalid number '--'
neovim-unwrapped> sh: invalid number '--'
neovim-unwrapped> CMake Error at /build/source/cmake/RunTests.cmake:98 (message):
neovim-unwrapped>   functional tests failed with error: Process terminated due to timeout
```

Reply:
I've tested this PR without your commit and I could run `make functionaltest` (though some tests fail but it has always been the case). Could you check it was the case for you ? and if yes revert the changes that broke it. I use make functionaltest locally to develop on neovim. Tests are too flaky to enable it for everyone but it's important to keep the ability to run them.

Reply:
Tests fail for me even on branch master :/